package.json

```diff
@@ -22,8 +22,13 @@
     "properties": {
       "url": {
         "type": "string",
-        "default": "http://examples/logo.png",
+        "default": "http://www.example.com/my-logo.png",
         "title": "Logo URL"
+      },
+      "style": {
+        "type": "string",
+        "default": "background: white;",
```

Comment:
Could you change `package.json:30` default value to `background: transparent;` or `background: none;` or empty string? GitBook is not fully white (it's actually #FAFAFA) so any semi transparent logo you might have will have a white box around it by default.

Especially in the Night mode, example:
![2018-03-21 19_30_22-introduction gitbook](https://user-images.githubusercontent.com/10331278/37732996-90e62158-2d3e-11e8-89d5-346a366f3599.png)

Reply:
Changing it to background: none;